src/DirectusProvider.tsx

```diff
@@ -17,12 +12,14 @@ export const DirectusProvider = ({ apiUrl, options, children }: DirectusProvider
     () => ({
       apiUrl: apiUrl,
       directus: new Directus(apiUrl, options),
-      DirectusAsset: (props: DirectusAssetPropsContextualized<DirectusAssetProps>): JSX.Element => (
```

Comment:
We are removing `DirectusAssetPropsContextualized` in favor of `DirectusAssetsProps`, so when using the "old" method you get the warning but you can still provide `apiUrl` while previously it was omitted. Can be this cause of problems?

Reply:
I doubt that it would cause any problems. As far as I understood it, the reason for DirectusAssetPropsContextualized was to make the apiUrl optional. However, with the recent changes, the apiUrl is now always optional.